app/src/main/java/com/imaginaryrhombus/proctimer/application/TimerRemoteConfigClientInterface.kt

```diff
@@ -0,0 +1,34 @@
+package com.imaginaryrhombus.proctimer.application
+
+/**
+ * Firebase RemoteConfig から情報を取得するクライアントのインターフェース.
+ */
+interface TimerRemoteConfigClientInterface {
+
+    /**
+     * RemoteConfig から情報を取得する.
+     * @param cacheExpireSeconds キャッシュが無効化される時間を設定.
+     * @param preApply 情報取得後、適用前に実行される.
+     * @param postApply 情報取得後、適用後に実行される.
+     * @note 情報取得に失敗した場合はどちらも実行されない.
+     */
+    fun fetchRemoteConfig(cacheExpireSeconds: Long = 60L,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fizxive/ProcTimer/blob/85662f57ce505e613f53a97ddbeebce37ccafc7f/app/src/main/java/com/imaginaryrhombus/proctimer/application/TimerRemoteConfigClientInterface.kt#L15"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>